spec/rubocop/cop/style/send_spec.rb

```diff
@@ -0,0 +1,43 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::Send do
+  subject(:cop) { described_class.new }
+
+  context 'with send' do
+    after(:each) { expect(cop.offenses.size).to eq(1) }
+
+    it 'registers an offense for a single expression' do
+      inspect_source(cop, 'send(:inspect)')
+    end
+
+    it 'registers an offense for a single expression without args' do
+      inspect_source(cop, 'send')
```

Comment:
Added a test for `send`, `public_send`, `__send__` without args, as it was something that I completely forgot that it could happen, during the initial PR code.